examples/default/example.tf

```diff
@@ -184,3 +184,22 @@ data "aws_iam_policy_document" "privileges" {
 output "ami" {
   value = "${data.aws_ami.ecs.id}"
 }
+
+# ------------------------------------------------------------------------------
+# ecs/spotfleet-cluster
```

Comment:
Create a new example and put a minimal example in there (basically only needs to launch the spotfleet cluster itself).